prime-dotnet-webapi/Controllers/SitesController.cs

```diff
@@ -1382,5 +1382,88 @@ public async Task<ActionResult> GetDeviceProviderSite(string deviceProviderId)
             var site = await _deviceProviderService.GetDeviceProviderSiteAsync(deviceProviderId);
             return Ok(site);
         }
+
+
+        // POST: api/Sites/5/archive
+        /// <summary>
+        /// Archive a site
+        /// </summary>
+        /// <param name="siteId"></param>
+        /// <param name="siteNoteUpdateModel"></param>
+        [HttpPost("{siteId}/archive", Name = nameof(ArchiveSite))]
+        [Authorize(Roles = Roles.EditSite)]
```

Comment:
These should match what is in `site-registration-actions.component.html`

Reply:
Sorry, what should match what?

Reply:
Requiring Super Admin role